src/components/LearningJourney/UpdatedOn.jsx

```diff
@@ -0,0 +1,17 @@
+import { Fragment } from "react";
+
+const UpdatedOn = ({ createdOn, updatedOn }) => {
+  return (
+    <div className="updated-on-container flex space-x-2">
+      <span className="text-sm">Published {createdOn}</span>
```

Comment:
We should format the date when displaying.
ex:  **new Date(createdOn).toLocaleString()**

Reply:
I have done this on my latest branch